src/video.cpp

```diff
@@ -2003,6 +2172,8 @@ namespace video {
             active_hevc_mode = 0;
           }
 
+          last_encoder_probe_supported_invalidate_ref_frames = encoder->details->supports_invalidate_ref_frames;
```

Comment:
We need to set this in the other paths that assign `chosen_encoder` too.

Reply:
Will look into it.